open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-api/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/event/VirtualizerTopicListener.java

```diff
@@ -33,11 +55,101 @@ public VirtualizerTopicListener(OpenMetadataTopicConnector ivInTopicConnector){
     @Override
     public void processEvent (String event){
         log.info("The following event is received: " + event);
+        ObjectMapper objectMapper = new ObjectMapper();
         try{
-            ivInTopicConnector.sendEvent(event);
+            TableContextEvent eventObject = objectMapper.readValue(event, TableContextEvent.class);
+            Map<String, String> views = viewGeneratorConnector.processInformationViewEvent(eventObject);
+            List<InformationViewEvent> viewEvents = generateViewEvents(eventObject, views);
+            for (InformationViewEvent item : viewEvents){
+                ivInTopicConnector.sendEvent(objectMapper.writeValueAsString(item));
+            }
         }catch (Exception e){
             log.error("Error in sending event: ", e);
         }
     }
 
+    /**
+     * notify Information View OMAS
+     *
+     * @param tableContextEvent it contains the information from json which is sent from Information View
+     * @param  createdViews the table definitions created in gaian
+     *
+     * @return list of events published to kafka topic
+     */
+    public List<InformationViewEvent> generateViewEvents(TableContextEvent tableContextEvent, Map<String, String> createdViews) {
+        List<InformationViewEvent> events = new ArrayList<>();
+        try {
+            if (createdViews == null || createdViews.isEmpty()) {
+                log.info("No views were created in Gaian, nothing to publish");
+            } else {
+                for (Map.Entry<String, String> entry : createdViews.entrySet()) {
+                    if (entry.getValue() == null) {
+                        log.info("No view was created for " + entry.getKey());
+                    } else {
+                        events.add(createInformationViewEvent(entry.getKey(), entry.getValue(), tableContextEvent));
+                    }
+                }
+                //sendNotification(events);
```

Comment:
log.debug message doesn't seems to match with the code.
And please remove commented code